bsp_diff/caas/device/intel/civ/host/vm-manager/0022-Inhibit-the-Host-suspend-activity.patch

```diff
@@ -0,0 +1,72 @@
+From 7b3ab114eb9e7df6f5ce31c79e5c0c10bb1be105 Mon Sep 17 00:00:00 2001
+From: Shwetha B <shwetha.b@intel.com>
+Date: Fri, 9 Jul 2021 11:41:52 +0530
+Subject: [PATCH 4/7] Inhibit the Host suspend activity
+
+This patch sets the sleep-inhibitor in the Host
+so that when the sleep-inhibitor service is started
+it inhibits the Host suspend activity from any user.
+This will stop host suspend only when the Android screen is on
+
+Tracked-On: OAM-97461
```

Comment:
OAM-97461 looks to be from Android 11.
Why is it that this is not part of Android 13 horizontal and we need to port here?
Is there a Jira ticket submitted for horizontal?